### PR TITLE
doc: Add CloudQuery for projects using Cobra

### PR DESCRIPTION
**Pull request by yevgenypats**
Hi 👋 

We are using Cobra so thought to add our project here for people's reference. We used to learn a lot from the examples here so thought to share back.

### CODE REVIEW

**Review by marckhouzam — CHANGES_REQUESTED** [review]
Thanks @yevgenypats! Great to see more cool projects using Cobra 🐍

You will need to sign the CLA before we can accept this contribution. Please see : https://github.com/spf13/cobra/pull/1742#issuecomment-1164651292

**Comment by yevgenypats** [comment]
Not sure Im able to sign it with this link - https://cla-assistant.io/spf13/cobra?pullRequest=1742 maybe something is off there? or should I try again a bit later?

**Comment by yevgenypats** [comment]
> Not sure Im able to sign it with this link - https://cla-assistant.io/spf13/cobra?pullRequest=1742 maybe something is off there? or should I try again a bit later?

Ok seems like the CLA was down :) signed now.

File: projects_using_cobra.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 - [Bleve](https://blevesearch.com/)
 - [CockroachDB](https://www.cockroachlabs.com/)
 - [Cosmos SDK](https://github.com/cosmos/cosmos-sdk)
+- [CloudQuery](https://github.com/cloudquery/cloudquery)
```
[the remark this answers is not in the document]
Comment (yevgenypats) [reply projects_using_cobra.md]: Done

**Comment by umarcor** [comment]
@marckhouzam please milestone this PR.

**Pull request merged**